app/retail/templates/email_unsubscribed.html

```diff
@@ -0,0 +1,52 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
```

Comment:
when i go to this page it advertises that i've been unsubscribed but doesnt actually flip the switch in the DB.  http://bits.owocki.com/8d5b49fe9d37/Screen%20Recording%202019-10-14%20at%2001.07%20PM.gif

if were saying its a one click unsubscribe then it probably should be